back/app/policies/idea_policy.rb

```diff
@@ -34,6 +34,7 @@ def index_mini?
   end
 
   def create?
+    return false if blocked?
```

Comment:
It appears we actually create an idea when a user submits a native survey, and line 40 was permitting this even when `user.active?` was `false`, so I added a check for `blocked?` first.

Reply:
Ticket to investigate possible bug created: https://citizenlab.atlassian.net/browse/CL-3212